### PR TITLE
Bootstrap

### PR DESCRIPTION
**Pull request by gammazero**
- Add bootstrap functionality.  This connects the indexer to configured peers to enable the exchange of gossip pubsub.
- Configure list of PubSub peers.  This allows the indexer to accept pubsub messges that _originate_ from these peers.   This is a temporary fix until go-legs can use the indexer's allow/deny policy to determine whether or not to accept a pubsub message.

### CODE REVIEW

File: config/bootstrap.go [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+)
+
+const (
+	defaultMinimumPeers     = 1
+	defaultBootstrapPeriod  = 30 * time.Second
+	defaulConnectionTimeout = defaultBootstrapPeriod / 3
+)
+
+// defaultBootstrapAddresses are the hardcoded bootstrap addresses.
+var defaultBootstrapAddresses = []string{}
```
Comment (willscott) [inline config/bootstrap.go:19]: are you planning to load this in via the `go build` or some other way?

**Review by willscott — APPROVED** [review]

File: config/bootstrap.go (hunk 1 above)
Comment (gammazero) [reply config/bootstrap.go:19]: No.  This is just a place holder for if/when we want to hardcode some bootstrap addresses as IPFS does.

Comment (willscott) [reply config/bootstrap.go:19]: we should find a way to default to the filecoin bootstrap nodes, probably

Comment (willscott) [reply config/bootstrap.go:19]: namely we should use https://github.com/filecoin-project/lotus/blob/7a55e8e892a2ed7a76ae4a75c1e33580688c3d0e/build/bootstrap/mainnet.pi

Comment (gammazero) [reply config/bootstrap.go]: Added hardcoded values.  These are written into the config file during `./storetheindex init`

**Pull request merged**